_data/simple-icons.json

```diff
@@ -5169,6 +5169,11 @@
             "hex": "34A853",
             "source": "https://sheets.google.com"
         },
+        {
+            "title": "Google Slides",
+            "hex": "FBBC04",
+            "source": "https://about.google/products/#all-products"
```

Comment:
Is this the source? I can only see PNG files there - whereas visiting https://slides.google.com gives me an SVG....

Reply:
Weird. I remember taking Google Forms svg from here.
Anyway, I have changed it now.